lib/travis/build/script/matlab.rb

```diff
@@ -0,0 +1,35 @@
+module Travis
+  module Build
+    class Script
+      class Matlab < Script
+        MATLAB_INSTALLER_LOCATION = 'https://ssd.mathworks.com/supportfiles/ci/ephemeral-matlab/v0/install.sh'.freeze
+        MATLAB_START = 'matlab -batch'.freeze
+        MATLAB_COMMAND = "results = assertSuccess(runtests('IncludeSubfolders',true))".freeze
```

Comment:
I think we can do without assigning to `results` and just have `MATLAB_COMMAND = "assertSuccess(runtests('IncludeSubfolders',true));".freeze` (with a semicolon at the end). If `assertSuccess` fails it throws an exception, in which case results won't be returned anyway. If it succeeds, results will be returned and displayed but that will be inconsistent with the failure case.

Reply:
Ohhh I see. That makes a lot of sense. I went ahead and updated the command to reflect that 😄 
(If you're wondering about the force-push.. I almost forgot the semicolon) 🤣